charts/agent-control/templates/installation-job.yaml

```diff
@@ -1,5 +1,7 @@
-{{- if .Values.agentControlCd.enabled }}
-{{- $installJobName := include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "flux-install-job" ) -}}
+{{- if (or .Values.agentControlDeployment.enabled .Values.agentControlCd.enabled) -}}
+{{- $installJobName := include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "install-job" ) -}}
+{{- $secretName := include "agent-control.secret.name" . -}}
+{{- $fluxSecretName := printf "%s-flux" (include "agent-control.secret.name" .) -}}
```

Comment:
could we follow "just" in case the same naming conventions we have for subAgents localData/values?
This in case in the future we start receiving the whole config and we need to do the classical local vs remote thing

Reply:
Thanks for noticing! I'll rework that